### ISSUE TITLE
[ Feature ] - Ability for the software to turn on automatically if you try to increase the brightness above regular limit

### ISSUE THREAD

**Issue by Tarik1700** [niklasr22/BrightIntosh#58]
Basically if you press the brightness up button even after your brightness is at max, then it should activate, if you lower it then it should de-activate.

**Comment by linodefdrytf** [comment]
Yeah it's a must have feature.On top menu bar show brightness to 100% and 160%,similar to what "Better Display"
![WhatsApp Image 2024-03-16 at 13 31 53](https://github.com/niklasr22/BrightIntosh/assets/160316647/1e83b505-7e9a-496d-8e74-491cf58cbb1c)
offers.Other wise it may increase too much battery consumption.
<img width="266" alt="Screenshot 2024-03-17 at 9 22 14 PM" src="https://github.com/niklasr22/BrightIntosh/assets/160316647/dbada890-74fc-4a94-a41a-c1d00a954dcc">
The above img shows rate of discharge when BrightIntosh is turned off(SDR) and the below when turned on(HDR).
<img width="256" alt="Screenshot 2024-03-17 at 9 24 03 PM" src="https://github.com/niklasr22/BrightIntosh/assets/160316647/234cb509-d2af-4926-af85-d2f143e220fb">
The display nits is fairly the same(i.e I reduced the brightness to match,not exact though).

**Comment by Gea97** [comment]
> As far as I know, this requires the use of private macOS APIs whose use is not App Store compliant

What about just adding a second slider for "normal" brightness? Does it also requires private macOS APIs?

I mean, it would be less annoying to have everything in just one Menu Bar Icon
